functions/convert/main.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+
+# Method for converting MD files to HTML
+
+from __future__ import print_function, unicode_literals
+
+import codecs
+import os
+import re
+import shutil
+import tempfile
+from contextlib import closing
+import markdown
+import boto3
+import datetime
+
+from glob import glob
+from general_tools.file_utils import unzip, add_file_to_zip, make_dir, write_file
+from general_tools.print_utils import print_error, print_ok
+from general_tools.url_utils import download_file
+
+event = {}
+context = {}
```

Comment:
It's not clear to me what purpose these global variables serve. They aren't used in `run(self)`, and are essentially used as local variables in `handle(..)` (i.e. they are written to before being read from). Further, they aren't used in other modules (at least as far as I can tell). Am I missing something, or can we get rid of these?